monitoring/understanding-the-monitoring-stack.adoc

```diff
@@ -9,18 +9,26 @@ toc::[]
 include::modules/monitoring-about-cluster-monitoring.adoc[leveloffset=+1]
 
 // Understanding the monitoring stack
-include::modules/monitoring-understanding-the-monitoring-stack.adoc[leveloffset=+1]
+include::modules/about-monitoring-stack-components.adoc[leveloffset=+1]
 include::modules/monitoring-default-monitoring-components.adoc[leveloffset=+2]
 include::modules/monitoring-default-monitoring-targets.adoc[leveloffset=+2]
 include::modules/monitoring-components-for-monitoring-user-defined-projects.adoc[leveloffset=+2]
+
+A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects].
+
 include::modules/monitoring-targets-for-user-defined-projects.adoc[leveloffset=+2]
 
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics, alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI. You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to perform the following tasks:
+
+* xref:../monitoring/managing-metrics.adoc#managing-metrics[Manage metrics]
+* xref:../monitoring/managing-alerts.adoc#managing-alerts[Manage alerts]
+* xref:../monitoring/reviewing-monitoring-dashboards.adoc#reviewing-monitoring-dashboards[Review monitoring dashboards]
+
+
+As a cluster administrator, you can find answers to common problems such as user metrics unavailability and Prometheus consuming a lot of disk space in xref:../monitoring/troubleshooting-monitoring-issues.adoc#troubleshooting-monitoring-issues[troubleshooting monitoring issues].
+
 == Additional resources
 
 * xref:../support/remote_health_monitoring/about-remote-health-monitoring.adoc#about-remote-health-monitoring[About remote health monitoring]
 * xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects_enabling-monitoring-for-user-defined-projects[Granting users permission to monitor user-defined projects]
 * xref:../security/tls-security-profiles.adoc#tls-security-profiles[Configuring TLS security profiles]
-
-== Next steps
```

Comment:
When the Monitoring book was restructured for 4.6, at the introduction of monitoring for user-defined projects, the assemblies were developed to build on each other sequentially. After reading about the concepts relating to the monitoring stack on this page, the intention was for the next step in the reader's journey to be "Configuring the monitoring stack".

The steps in the "Configuring the monitoring stack" page must be reviewed and configured according to the customer's needs before the procedures in the remaining assemblies. My view is that the "Next steps" section provides a clear direction as to what should be done next, then the "Additional resources" section provides further reading on topics related to the material in this page. "Next steps" and "Additional resources" sections are consistent with other pages in this book and in the library.

What is the underlying requirement behind removing this or renaming it?

Reply:
Yeah given the context , I have added back the next steps.